pkgs/development/libraries/protobuf/21.nix

```diff
@@ -0,0 +1,6 @@
+{ callPackage, abseil-cpp, ... }:
+
+callPackage ./generic-v3-cmake.nix {
```

Comment:
Is `v3` here still correct/relevant?

Reply:
we can probably remove it. shouldn't be meaningful anymore.